### PR TITLE
Review advanced

### PR DESCRIPTION
**Pull request by AnnikaLau**

### CODE REVIEW

File: advanced/Exercise_3_gitignore.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ In this exercise, you will first use a provided script to generate a simple Git
 
 * [Generate the README file in HTML](#html)
```
Comment (mjaehn) [inline advanced/Exercise_3_gitignore.md:9]: Please check for consistency for the whole exercise: README vs _README_

File: advanced/Exercise_5_git-rebase.md [hunk 2]
```diff
@@ -95,19 +95,27 @@ Add and commit this change.
...
-Examine the repository status and log again. Even though we did a merge command, by doing the rebase beforehand, we have created a situation where the merge can be done using the fast-forward technique, and therefore no merge commit was required. This is particularly advantageous in actively developed codes because it allows the history of the *main* branch to remain clean and free of unnecessary merge commits.
+Examine the repository status and log again. Even though we did a merge command, by doing the rebase beforehand, we have created a situation where the merge can be done using the fast-forward technique, and therefore no merge commit was required. This is particularly advantageous in actively developed codes because it allows the history of the `main` branch to remain clean and free of unnecessary merge commits.
+
+Don't forget to delete the `rebase_feature` branch after this merge.
+
+```plaintext
+git branch -d rebase_feature
+```
+
+To visualise the difference, take a look at the Git log graph (remember what we learned in [Exercise 1](./Exercise_1_git-log-diff-show.md#use-git-log-to-have-a-look-at-the-repository))
```
Comment (mjaehn) [inline advanced/Exercise_5_git-rebase.md:121]: I think we use American English (visuali**z**e) for most of the rest. Can be considered as another consistency check.

File: advanced/Exercise_8_git-submodule.md [hunk 3]
```diff
@@ -138,9 +138,10 @@ Finally, let's learn how to pull changes from the submodule into our host reposi
 
 Navigate back to your fork of the *posters* repository on GitHub, and make a change to the schedule file, and commit it. You can do this directly in the web interface by simply selecting the file and using the edit button.
 
-Once you have done this, go to back to the terminal and use `git submodule update` to get the main repository to fetch the new commit from the *posters* repository.
+Once you have done this, navigate back into into root of the *conference_planning* folder and use `git submodule update` to get the main repository to fetch the new commit from the *posters* repository.
```
Comment (mjaehn) [inline advanced/Exercise_8_git-submodule.md:141]:
```suggestion
Once you have done this, navigate back to the root of the *conference_planning* folder and use `git submodule update` to get the main repository to fetch the new commit from the *posters* repository.
```

**Review by mjaehn — CHANGES_REQUESTED** [review]
Dear @AnnikaLau, fantastic work on the revision of the 2024 advanced course! Especially the overall formatting and the overhaul of the _worktree_ section. 

There are just a few minor comments regarding language and consistency. Otherwise, this is ready to be merged. 👍🏻

**Comment by AnnikaLau** [comment]
> Dear @AnnikaLau, fantastic work on the revision of the 2024 advanced course! Especially the overall formatting and the overhaul of the _worktree_ section.
> 
> There are just a few minor comments regarding language and consistency. Otherwise, this is ready to be merged. 👍🏻

Thank you @mjaehn for your fantastic review! I implemented all your requested changes.

**Review by mjaehn — APPROVED** [review]
LPTM!

**Pull request merged**